docs/src/modules/nixos/desktop.md

```
@@ -33,4 +37,4 @@ This module is a base component for desktop hosts. It must be manually imported
 ## Operational Notes
 
 - This module ensures that all desktop hosts have a consistent baseline for graphical interfaces and remote management.
-- If you need to disable a specific component imported by this module, you may need to use `lib.mkForce` or target the specific component's enable option if available.
+- If you need to disable a specific component imported by this module, you may need to use `lib.mkForce` or target the specific component's enable option if available.
```

markdownlint-cli2 (0.22.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)